JustReleaseNotes/command_line.py

```diff
@@ -47,7 +47,13 @@ def generate_release_notes(configFile):
             directory = os.path.join(releaseNotesConfig["pathToSave"],packageName)
             conf["Source"]["Directory"] = directory
             repo = JustReleaseNotes.sourcers.factory.create(conf["Source"])
-            writer = JustReleaseNotes.writers.factory.create(conf["ReleaseNotesWriter"], ticketProvider)
+
+            writerTemplate = "${content}"
+            if "ContentTemplateFile" in conf:
```

Comment:
Is `ContentTemplateFile` in the main config or the section for the writer ? Theoretically the template is just for the writer so maybe it should live in the writer section

But then the question is - if we had multipole writers would we have to specify template for each of them or would they be using the same template?